### PR TITLE
repo: Update dev/build/deploy README

### PR DESCRIPTION
**Pull request by phlax**
Signed-off-by: Ryan Northey <ryan@synca.io>

### CODE REVIEW

**Comment by phlax** [comment]
current README rendering is here https://github.com/envoyproxy/envoy-website/blob/cc37e2d1891d71c816b827eff2a4923c98fa97fb/README.md

File: README.md [hunk 1]
```diff
@@ -1,59 +1,124 @@
...
+
+```
 
 ### Site content
 
 Item | Path
 :----|:----
 Home page content | [`_data/home.yml`](./_data/home.yml)
 Nav links on the main page | [`_data/nav.yml`](./_data/nav.yml)
+Documentation versions | [`_data/versions.yaml`](./_data/versions.yml)
```
Comment (phlax) [inline README.md:121]: @mattklein123 @alyssawilk @RyanTheOptimist just flagging this change to the release process...

where we previously edited `_config.yml` now we need to edit `_data/versions.yml`

File: README.md [hunk 2]
```diff
@@ -1,59 +1,124 @@
...
+```
 
-### Deploying to Github Pages
+From there you can run bazel commands directly, for example:
 
-To deploy your changes all you have to do is push to `master` and Github pages will automatically run `jekyll build` and
-deploy the generated files.
+```console
+
+me@73deecbfaf55:/src/workspace/envoy-website$ bazel run //docs:latest_version
```
Comment (kfaseela) [inline README.md:109]: may be the console prompt can be a bit better? :)

**Review by kfaseela — COMMENTED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (phlax) [reply README.md]: changed `me@` -> `username@`

**Pull request merged**